inference-engine/tests/functional/plugin/cpu/single_layer_tests/activation.cpp

```diff
@@ -58,6 +58,7 @@ class ActivationLayerCPUTest : public testing::WithParamInterface<ActivationLaye
         std::tie(basicParamsSet, cpuParams) = this->GetParam();
 
         std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
+        configuration.insert({"DUMP_CONSTANT_NODES", CONFIG_VALUE(YES)});
```

Comment:
Just curious, why do we want it to be enabled by default in tests?
If there is the reason to do so, instead of updating every test, can we just add:
```c++
if (targetDevice = "CPU")
    configuration.insert({"DUMP_CONSTANT_NODES", CONFIG_VALUE(YES)});
```
To LoadNetwork from layer_test_utils.cpp

Reply:
We need all the nodes, otherwise we will get an error:
https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/plugin/cpu/test_utils/cpu_test_utils.cpp#L138

Reply:
is it possible to make that check slightly more complex so we could avoid other changes?

Reply:
@egorshul we shouldn't update all tests with logic like this: `configuration.insert({"DUMP_CONSTANT_NODES", CONFIG_VALUE(YES)});`. Lets do this in base class.